packages/typespec-ts/test/azureModularIntegration/routes.spec.ts

```diff
@@ -0,0 +1,229 @@
+import { RoutesClient } from "./generated/routes/src/index.js";
+import { assert } from "chai";
+describe("Routes Client", () => {
+    let client: RoutesClient;
+
+    beforeEach(() => {
+        client = new RoutesClient({
+            allowInsecureConnection: true,
+            endpoint: "http://localhost:3002"
+        });
+    });
+
+    it("Routes_InInterface", async () => {
+        const result = await client.inInterface.fixed();
+        assert.isUndefined(result);
```

Comment:
@v-jiaodi Could we simplify the case with below? I don't think there is any value to assert undefined.

```ts
 it("Routes_InInterface", async () => {
        await client.inInterface.fixed();
});
```